calico_node/filesystem/etc/calico/confd/config/custom_filters.cfg

```diff
@@ -0,0 +1,5 @@
+# Generated by confd
```

Comment:
@neiljerram here's what I'm talking about (errors stopped when I created needed directories in etcd):
http://pastebin.com/Btu2Ethf

So without placeholders that would cause error in BIRD since `/etc/calico/confd/config/custom_filters.cfg` and `/etc/calico/confd/config/custom_filters6.cfg` do not exist.